src/frontend/komponenter/Modal/ModalWrapper.tsx

```diff
@@ -62,27 +43,27 @@ export const ModalWrapper: React.FC<ModalProps> = ({
                 aria-label={ariaLabel ? ariaLabel : tittel || ''}
                 header={tittel ? { heading: tittel, closeButton: !!onClose } : undefined}
             >
-                <Modal.Body>
-                    <Innhold>{children}</Innhold>
-                    {aksjonsknapper && (
-                        <ButtonContainer $marginTop={aksjonsknapper.marginTop}>
-                            <ModalKnapp
-                                variant="tertiary"
-                                onClick={aksjonsknapper.lukkKnapp.onClick}
-                                disabled={aksjonsknapper.lukkKnapp.disabled}
-                            >
-                                {aksjonsknapper.lukkKnapp.tekst}
-                            </ModalKnapp>
-                            <ModalKnapp
-                                variant="primary"
-                                onClick={aksjonsknapper.hovedKnapp.onClick}
-                                disabled={aksjonsknapper.hovedKnapp.disabled}
-                            >
-                                {aksjonsknapper.hovedKnapp.tekst}
-                            </ModalKnapp>
-                        </ButtonContainer>
-                    )}
-                </Modal.Body>
+                <Modal.Body>{children}</Modal.Body>
```

Comment:
Når det er en slik endring på en felleskomponent kan det være fint å ha med et bilde for å se hvordan det påvirker andre som bruker denne

Reply:
På MAIN:
<img width="1695" alt="Skjermbilde 2024-05-30 kl  11 11 53" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/28704275/458dfad8-2528-4e06-b175-240569a98460">


I denne branchen:
<img width="1323" alt="Skjermbilde 2024-05-30 kl  11 10 44" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/28704275/3e748499-144c-4ebf-beed-eac9d80b4fe8">

Reply:
Main: 

<img width="1401" alt="Skjermbilde 2024-05-30 kl  12 24 14" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/28704275/615669d4-6dc6-44fb-bc68-86d16720c79f">

På branch:
<img width="1485" alt="Skjermbilde 2024-05-30 kl  12 20 54" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/28704275/1225d683-a2d0-4b43-8e11-66e8e164d868">

Reply:
PÅ MAIN:

<img width="1208" alt="Skjermbilde 2024-05-30 kl  12 53 02" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/28704275/02b7d6f6-079b-4b42-9c57-27355eaedbb8">


PÅ BRANCH:

<img width="1089" alt="Skjermbilde 2024-05-30 kl  12 53 27" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/28704275/4eb3bde7-d80c-408e-aac6-ce8b673b5c02">

Reply:
PÅ MASTER:
<img width="786" alt="Skjermbilde 2024-05-30 kl  13 02 59" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/28704275/966c9a94-4677-47c9-918d-6ff2af3dbec5">

PÅ BRANCH:

<img width="751" alt="Skjermbilde 2024-05-30 kl  13 01 47" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/28704275/ae8249e1-26db-4407-be66-1019baf9b94f">

Reply:
Marie tok en kikk på disse screenshotsene, og syns endringene så fine ut